apps/explorer/src/hooks/useImageMod.ts

```diff
@@ -1,7 +1,22 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
+import { useAppsBackend } from '@mysten/core';
 import { useQuery } from '@tanstack/react-query';
 
+export enum VISIBILITY {
+    PASS = 'PASS',
+    BLUR = 'BLUR',
+    HIDE = 'HIDE',
+}
+
+type ImageModeration = {
+    visibility?: VISIBILITY;
+};
+
+const placeholderData = {
+    visibility: VISIBILITY.PASS,
```

Comment:
I think we should default to false if we can't moderate an image - or at the very least to blur, so that we don't accidentally show something we're not supposed to

Reply:
I think we fail open intentionally, actually.

Reply:
ah yea, I thought in a previous discussion we talked about failing to a blur / block state, but looking through previous logic we default to `pass` for failed checks